modules/view/module.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": [
-    "selection"
+    "selection",
+    "component"
```

Comment:
Not sure how to add 'test dependencies' - I've replicated the error in the issue as a test and it uses `.api()` which is part of `component`